executable/mach_o.ksy

```diff
@@ -730,6 +754,72 @@ types:
       - id: release
         type: u1
     -webide-representation: '{major:dec}.{minor:dec}'
+  encryption_info_command:
+    seq:
+      - id: cryptoff
+        type: u4
+      - id: cryptsize
+        type: u4
+      - id: cryptid
+        type: u4
+      - id: pad
+        type: u4
+        if: _root.magic == magic_type::macho_be_x64 or _root.magic == magic_type::macho_le_x64
+  twolevel_hints_command:
+    seq:
+      - id: offset
+        type: u4
+      - id: nhints
+        type: u4
+  linker_option_command:
+    seq:
+      - id: count
+        type: u4
```

Comment:
BTW, according to the [style guide](http://doc.kaitai.io/ksy_style_guide.html) we're trying to introduce, it's generally suggested that it's better to call this `num_strings`, to maintain the same consistent naming style everywhere.